lib/ably/exceptions.rb

```diff
@@ -13,9 +13,9 @@ module Exceptions
     # @!attribute [r] code
     #   @return [String] Ably specific error code
     class BaseAblyException < StandardError
-      attr_reader :status, :code
+      attr_reader :status, :code, :request_id
 
-      def initialize(message, status = nil, code = nil, base_exception = nil, options = {})
+      def initialize(message, status = nil, code = nil, base_exception = nil, options = {}, request_id = nil)
```

Comment:
I would have preferred for `request_id` to be an option given it's a flexible hash for this reason.

Reply:
Didn’t know the supposed usage of that hash, that’s why I didn’t touch it. Good to know that I can use it.